src/sandstorm/sandstorm-http-bridge.c++

```diff
@@ -784,23 +784,41 @@ private:
 #undef ON_EVENT
 
   static void parseETag(kj::StringPtr input, WebSession::ETag::Builder builder) {
+    auto maybePrintInvalidEtagWarning = [](kj::StringPtr input) -> void {
```

Comment:
It's pretty unusual to use a lambda in this way, and it has a (small) runtime cost since you're actually creating a stack-allocated value that holds this lambda. I would instead make this a static method, sibling to `parseETag()`.